manifests/prometheus/spec/alerts/aiven-cost.test.yml

```diff
@@ -0,0 +1,24 @@
+---
+rule_files:
+  # See alerts_validation_spec.rb for details of how this gets set:
+  - spec/alerts/fixtures/rules.yml
+
+evaluation_interval: 1h
+
+tests:
+  - interval: 1h
```

Comment:
I think I would add a second test here that shows that it doesn't fire when ..
1) there's enough data to make a projection, and 
2) the projection doesn't go above the threshold

The tests at the moment only show it doesn't fire when there's not enough data, and when the condition is met.

Reply:
Yep - will add one.

Reply:
✅

Reply:
Tests looks good :+1: I might be tempted to swap the last two around, because the convention established in other tests is
1. Not enough data
2. Enough data but not passing the condition
3. Enough data and passing the condition

It doesn't matter that much though, so I'll approve it and merge it in the morning.